src/blockdata/script.rs

```diff
@@ -51,9 +53,79 @@ use crate::schnorr::{TapTweak, TweakedPublicKey, UntweakedPublicKey};
 /// ### Bitcoin Core References
 ///
 /// * [CScript definition](https://github.com/bitcoin/bitcoin/blob/d492dc1cdaabdc52b0766bf4cba4bd73178325d0/src/script/script.h#L410)
-#[derive(Clone, Default, PartialOrd, Ord, PartialEq, Eq, Hash)]
-pub struct Script(Box<[u8]>);
+#[derive(PartialOrd, Ord, PartialEq, Eq, Hash)]
+pub struct Script([u8]);
```

Comment:
As you work on this perhaps include this doc string modified from stdlib

```
+/// This is an unsized type, meaning that it must always be used behind a pointer like & or Box. For
+/// an owned version of this type, see [`ScriptBuf`].
```

Reply:
Yeah, doc will be improved when I get more concept ACKs since there are two lines of `unsafe`.

Reply:
no worries, just thought I'd throw it in since I was reading up trying to figure out what you were doing :)